server/plugin/mm_34646_token_refresh.go

```diff
@@ -0,0 +1,125 @@
+package plugin
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/google/go-github/v31/github"
+	"github.com/pkg/errors"
+)
+
+const pageSize = 100
+const delayBetweenPages = 5 * time.Minute
+
+func (p *Plugin) forceResetAllMM34646() error {
+	config := p.getConfiguration()
+	ctx := context.Background()
+
+	for page := 0; ; page++ {
+		keys, appErr := p.API.KVList(page, pageSize)
+		if appErr != nil {
+			return appErr
```

Comment:
Should we _continue_ here (like below)?

Reply:
1/5 this is a "bad" error, so we should fail and bail out. Below are mostly expected conditions (like skipping keys we don't understand). I can see the argument for either bailing on KV.Get/Set appErrs below.

Reply:
I was approaching this from the angle of a "KV" operation failing for some reason (where KVGet fails for a key we continue, if KVList fails for a page we bail out).  I can see the argument where a page failure is more impactful than a per-key failure (though on the extreme we could get all key failures). 0/5 on this, just wanted to point the diff. handling.

Reply:
Given the redundant nature of this script, `continue` is fine, will change.

Reply:
Come to think of it, we'd  return upon `KVList` failure anyway (`len(keys) < pageSize)`) sp keeping as is.